docs/setup/storage/s3.md

```diff
@@ -11,49 +11,115 @@ has_children: false
 # Prepare Your AWS S3 Bucket
 
 1. From the S3 Administration console, choose `Create Bucket`.
-1. Make sure that you:
+2. Make sure that you:
    1. Block public access.
-   1. Disable Object Locking.
-1. lakeFS requires permissions to interact with your bucket. The following is a minimal bucket policy. To add it, go to the `Permissions` tab, and paste it as :
-
-   ```json
-   {
-     "Id": "Policy1590051531320",
-     "Version": "2012-10-17",
-     "Statement": [
-       {
-         "Sid": "Stmt1590051522178",
+   2. Disable Object Locking.
```

Comment:
1. "Object Lock"
2. Think you need to have link for both terms to the aws website